locale/fr_FR/LC_MESSAGES/django.po

```diff
@@ -320,12 +328,17 @@ msgstr "<a href=\"%(path)s/editions\">%(count)s éditions</a>"
 #: bookwyrm/templates/book/book.html:142
 #, python-format
 msgid "This edition is on your <a href=\"%(path)s\">%(shelf_name)s</a> shelf."
-msgstr "Cette édition est sur votre étagère <a href=\"%(path)s\">%(shelf_name)s</a>."
+msgstr ""
+"Cette édition est sur votre étagère <a href=\"%(path)s\">%(shelf_name)s</a>."
 
 #: bookwyrm/templates/book/book.html:148
 #, python-format
-msgid "A <a href=\"%(book_path)s\">different edition</a> of this book is on your <a href=\"%(shelf_path)s\">%(shelf_name)s</a> shelf."
-msgstr "Une <a href=\"%(book_path)s\">édition différente</a> de ce livre existe sur votre étagère <a href=\"%(shelf_path)s\">%(shelf_name)s</a>."
+msgid ""
+"A <a href=\"%(book_path)s\">different edition</a> of this book is on your <a "
+"href=\"%(shelf_path)s\">%(shelf_name)s</a> shelf."
+msgstr ""
+"Une <a href=\"%(book_path)s\">édition différente</a> de ce livre existe sur "
+"votre étagère <a href=\"%(shelf_path)s\">%(shelf_name)s</a>."
```

Comment:
Is the single line causing any problem?

Splitting makes it harder to search for strings when translating because one can’t know for sure at what part the string is split and searching for a string becomes unreliable.

Reply:
This question also applies on following splits.

Reply:
The PR includes a lot of those splits so I’ll continue the review once we resolved this question.

Reply:
I agree with you that it's less practical but the line cuts have been made automatically (I don't know what caused it, AFAIK it's not my doing)

Reply:
Oh shoot… The line split was originally there and was removed after we discussed the topic.

@mouse-reeve: Do you have any insight on this change?

Reply:
I can't do it right now, but I'll check out later if it was caused by aggressive linting when I wrote the changes to the file (it was on another branch)

Reply:
Ok, I fixed it

Reply:
Probably my bad, I've been having trouble getting `makemessages` to work properly on my laptop and accidentally ran it with the wrong flags a few times. Glad it's fixed!

Reply:
Thanks @joachimesque. :)